src/homepageExperience/components/DataListening.tsx

```diff
@@ -0,0 +1,181 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {RouteComponentProps, withRouter} from 'react-router-dom'
+
+// Apis
+import {runQuery} from 'src/shared/apis/query'
+
+// Components
+import {ErrorHandling} from 'src/shared/decorators/errors'
+import ConnectionInformation, {LoadingState} from './ConnectionInformation'
+import {Button} from '@influxdata/clockface'
+
+interface OwnProps {
+  bucket: string
+}
+
+interface State {
+  loading: LoadingState
+  timePassedInSeconds: number
+  secondsLeft: number
+  previousBucket: string
+  retry: boolean
+}
```

Comment:
This file was also reused, with minor changes: 

1. The Listen for data button was removed.
2. A retry button was added, for when the request times out, the user can retry.